pkg/controller/controller.go

```diff
@@ -163,6 +172,27 @@ func (c *ctrl) onFunctionAddedOrUpdated(function *v1.Function) {
 	}
 }
 
+func (c *ctrl) onFunctionUpdated(oldFn *v1.Function, newFn *v1.Function) {
+	if oldFn.Name != newFn.Name {
```

Comment:
I'm pretty sure this can't happen, per design, no? An "update" for k8s only happens if you're changing the same object, which is identified by its name+namespace

Reply:
Agreed. It's really an assertion since the behaviour is so predicated on the name and the namespace not changing.